users/nazu/keyrecords/process_records.h

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2023 Tom Sennewald (@beelzebubi)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#pragma once
+#include "nazu.h"
+
+enum userspace_custom_keycodes {
+    KC_QWERTY = SAFE_RANGE, // change to QWERTY layer
```

Comment:
Just a heads up, there is now a user and keyboard block for keycodes (each has 256 keycodes).  SAFE_RANGE starts at the kb block, and may collide with keycodes for the keyboard. 

```suggestion
    KC_QWERTY = QK_USER,    // change to QWERTY layer
```